src/pps.js

```diff
@@ -0,0 +1,58 @@
+import { auctionManager } from './auctionManager.js';
```

Comment:
since this is quite a bit of code I'm wondering if it should be in a module (maybe gptPreAuction) - what do you think @patmmccann ?